tests/adapter/dbt/tests/adapter/python_model/test_spark.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+PANDAS_MODEL = """
+import pandas as pd
+
+def model(dbt, session):
+    dbt.config(
+        materialized="table",
+    )
+
+    df = pd.DataFrame(
+        {'City': ['Buenos Aires', 'Brasilia', 'Santiago', 'Bogota', 'Caracas'],
+        'Country': ['Argentina', 'Brazil', 'Chile', 'Colombia', 'Venezuela'],
+        'Latitude': [-34.58, -15.78, -33.45, 4.60, 10.48],
+        'Longitude': [-58.66, -47.91, -70.66, -74.08, -66.86]}
+        )
+
+    return df
+"""
+PYSPARK_MODEL = """
+# models/pyspark_df.py
+
+def model(dbt, session):
```

Comment:
Maybe drop the first two lines so all three models have the same format?
```suggestion
def model(dbt, session):
```